### PR TITLE
correct classlist toggle translation

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/domtokenlist/toggle/index.md [hunk 1]
```diff
@@ -4,58 +4,76 @@ slug: Web/API/DOMTokenList/toggle
...
 
-```plain
-tokenList.toggle(token, force);
+```js
+toggle(token);
+toggle(token, force);
 ```
 
-### 参数列表
+### 参数
```
Comment (yin1999) [inline files/zh-cn/web/api/domtokenlist/toggle/index.md:16]: ?

see line 23

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/domtokenlist/toggle/index.md [hunk 2]
```diff
@@ -4,58 +4,76 @@ slug: Web/API/DOMTokenList/toggle
...
+    span.textContent = `'c' removed; classList is now "${classes}".`;
   }
-}
+})
 ```
 
-The output looks like this:
+输出如下所示，并且每次点击文本时都会发生变化：
+
+{{ EmbedLiveSample('Toggling_a_class_on_click', '100%', 60) }}
```
Comment (yin1999) [inline files/zh-cn/web/api/domtokenlist/toggle/index.md:63]: please see <https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples>. You should update the param when translate those head. It's also metioned in [translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)

```suggestion
{{ EmbedLiveSample('在点击后 toggle class', '100%', 60) }}
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/domtokenlist/toggle/index.md [hunk 3]
```diff
@@ -4,58 +4,69 @@ slug: Web/API/DOMTokenList/toggle
...
-  - : 一个{{domxref("Boolean")}} 值，设置后会将方法变成单向操作。如设置为`false`, 则会删除标记列表中匹配的给定标记，且不会再度添加。如设置为 `true`, 则将在标记列表中添加给定标记，且不会再度删除。
+- `token`
+  - : 一个字符串，表示你想要 toggle 的标记。
+- `force` {{optional_inline}}
+  - : 如果包含该值，设置后会将方法变成单向操作。如果设置为 `false`，*仅*会删除标记列表中匹配的给定标记，且不会再度添加。如果设置为 `true`，*仅*在标记列表中添加给定标记，且不会再度删除。
 
 ### 返回值
 
-该方法返回一个{{domxref("Boolean")}}值 — 如给定标记不存在于列表中返回`false` , 标记存在则返回`true` 。
+一个布尔值，`true` 或者 `false`，表示在调用 toggle 后，标记是否存在与列表中。
```
Comment (yin1999) [inline files/zh-cn/web/api/domtokenlist/toggle/index.md:25]:
```suggestion
一个布尔值，`true` 或者 `false`，表示在调用 toggle 后，标记是否存在于列表中。
```

File: files/zh-cn/web/api/domtokenlist/toggle/index.md [hunk 4]
```diff
@@ -4,58 +4,69 @@ slug: Web/API/DOMTokenList/toggle
...
 ```js
-var span = document.querySelector("span");
-var classes = span.classList;
-span.onclick = function() {
-  var result = classes.toggle("c");
-  if(result) {
-    span.textContent = "'c' added; classList is now '" + classes + "'.";
+const span = document.querySelector("span");
+const classes = span.classList;
+span.addEventListener('click', () => {
```
Comment (yin1999) [inline files/zh-cn/web/api/domtokenlist/toggle/index.md:44]: Please copy those code from github editor or from mdn website. Do not copy it from github markdown preview (which may omit those empty line).

```suggestion
const classes = span.classList;

span.addEventListener('click', () => {
```

File: files/zh-cn/web/api/domtokenlist/toggle/index.md [hunk 5]
```diff
@@ -4,58 +4,69 @@ slug: Web/API/DOMTokenList/toggle
...
+  const result = classes.toggle("c");
+  if (result) {
```
Comment (yin1999) [inline files/zh-cn/web/api/domtokenlist/toggle/index.md:46]:
```suggestion
  const result = classes.toggle("c");

  if (result) {
```

File: files/zh-cn/web/api/domtokenlist/toggle/index.md (hunk 2 above)
Comment (Jack-Zhang-1314) [reply files/zh-cn/web/api/domtokenlist/toggle/index.md]: I see,thank you

**Review by yin1999 — COMMENTED** [review]
typo and style

**Review by yin1999 — APPROVED** [review]

**Pull request merged**